doc/languages-frameworks/haskell.section.md

```diff
@@ -644,7 +739,7 @@ haskellPackages.haskell-ci.overrideScope (self: super: {
 
 The custom interface comes into play when you want to override the arguments
 passed to `haskellPackages.mkDerivation`. For this, the function `overrideCabal`
-from `haskell.lib.compose` is used. E.g. if you want to install a man page
+from `haskell.lib.compose` is used. E.g., if you want to install a man page
```

Comment:
Commas before an if are rarely necessary even if every German instinct tells you they are!

```suggestion
from `haskell.lib.compose` is used. E.g. if you want to install a man page
```

Reply:
My languagetool always claims that commata are needed after "e.g."

Reply:
seems it read a different punctuation guide to myself :)

Reply:
After consulting the internet my decision seems to not be completely wrong.